.eslintrc

```diff
@@ -22,7 +22,7 @@
         "eqeqeq": 2,
         "eol-last": 2,
         "indent": [2, 4],
-        "linebreak-style": [2, "unix"],
+        "linebreak-style": [1, "unix"],
```

Comment:
Re: your comment in the issue about Windows and Git… can you explain a bit more of what's happening now? Is there a way to have a Windows environment that uses unix line endings?